lib/views/pages/account_settings/account_settings.dart

```diff
@@ -0,0 +1,174 @@
+                topRight: Radius.circular(30.0),
+                topLeft: Radius.circular(30.0)),
+            color: Colors.white,
+          ),
+          alignment: Alignment.center,
+          child: Container(
+            padding: const EdgeInsets.symmetric(vertical: 39, horizontal: 15),
+            child: Column(
+              children: [
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    IconButton(
+                        icon: const Icon(Icons.edit),
+                        onPressed: () {},
+                        color: Colors.transparent),
+                    Text(
+                      first_name + ' ' + last_name,
+                      style:
+                          TextStyle(fontSize: 20, fontWeight: FontWeight.w600),
+                    ),
+                    IconButton(
+                      icon: const Icon(Icons.edit),
+                      onPressed: () {
+                        print('clicked');
+                      },
+                      color: Color(0xFFA7ABB8),
+                      iconSize: 15.0,
+                      alignment: Alignment.centerLeft,
+                      padding: new EdgeInsets.all(5.0),
+                    ),
+                  ],
+                ),
+                Text(
+                  email,
+                  style: TextStyle(
+                      fontSize: 14,
+                      color: Color(0xFF324ED9),
+                      fontWeight: FontWeight.w500),
+                ),
+                Rectangle(
+                  content: Row(
+                    children: [
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            'Currency',
+                            style: TextStyle(
+                              fontSize: 12,
+                              fontWeight: FontWeight.w500,
+                              color: Color(0xFFA7ABB8),
+                            ),
+                          ),
+                          Text(
+                            currency,
+                            style: TextStyle(
+                                fontSize: 18,
+                                fontWeight: FontWeight.w600,
+                                height: 1.8),
+                          ),
+                        ],
+                      ),
+                      Spacer(),
+                      IconButton(
+                        icon: const Icon(Icons.edit),
+                        onPressed: () {
+                          print('clicked');
+                        },
+                        color: Color(0xFFA7ABB8),
+                        iconSize: 15.0,
+                        alignment: Alignment.centerRight,
+                        padding: new EdgeInsets.all(0.0),
```

Comment:
remove new keyword since it's not preferred ([reference](https://dart-lang.github.io/linter/lints/unnecessary_new.html))